woocommerce/wc-admin
# Add option remote inbox notifications rule processor

This adds a remote inbox notifications rule processor that enables looking at any option, as required [here](https://teamisotope.wordpress.com/2020/07/02/remote-inbox-notifications/#comment-4342)

### Screenshots
![image](https://user-images.githubusercontent.com/224531/94383619-ec326000-0183-11eb-976a-0f5ef8c5b94a.png)

### Detailed test instructions:
- Add this URL to `DataSourcePoller::DATA_SOURCES`: `https://gist.github.com/becdetat/7b0de142b0e708354d546618496a1d26/raw/249a0a653d894443ef9e65d69ac687561a2bd730/option-test.json`
- Set the store currency to something other than USD
- Run the `wc_admin_daily` cron task
- The "This should appear in USD stores" note should _not_ appear
- Set the store currency to USD
- Run the `wc_admin_daily` cron task
- The "This should appear in USD stores" note should now appear

### Changelog Note:

Development: Add remote inbox notifications rule allowing access to any option
